Commands/Anime/slap.js

```diff
@@ -11,37 +11,11 @@ module.exports = {
     },
     run: async(client, message, args) => {
         const { url } = await slap().catch(() => {});
-
         if (!url) return message.channel.send(`Could not connect to nekos.life`);
-    
         const embed = new MessageEmbed();
-    
-        if (
-          message.mentions.members.size &&
-          message.mentions.members.first().id === client.user.id
-        ) {
-          return message.channel.send(
-            `${
-              [`Ouch! How dare you slap me!`, `Stop that!`, `It hurts! ;-;`][
-                Math.floor(Math.random() * 2)
-              ]
-            }`
-          );
-        } else if (
-          message.mentions.members.size &&
-          message.mentions.members.first().id === message.author.id
-        ) {
-          return message.channel.send(`Wai~ Seriously!?`);
-        } else if (message.mentions.members.size) {
-          return message.channel.send(
-            embed
-              .setColor(config.embedcolor)
-              .setDescription(`${message.member} slapped ${message.mentions.members.first()}!`)
-              .setImage(url)
-          );
-        } else {
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`${[`Ouch! How dare you slap me!`, `Stop that!`, `It hurts! ;-;`][Math.floor(Math.random() * 2)]}`);
+         if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`Wai~ Seriously!?`); 

```

Comment:
You should use 
```js
message.channel.send({content: ""})
```

Reply:
If it's string only, you can use message.channel.send("string")